testing/sdk_marathon.py

```diff
@@ -41,6 +41,14 @@ def destroy_app(app_name):
     shakedown.delete_app_wait(app_name)
 
 
+def restart_app(app_name):
+    sdk_utils.out("Restarting {}...".format(app_name))
```

Comment:
House style seems to be double quotes, but ultimately I agree that we should just add style checks rather than nitpicking in PRs unless it's egregiously unreadable.